.github/workflows/test.yml

```diff
@@ -213,3 +213,27 @@ jobs:
         run: |
           make test-integration
         if: "env.GIT_DIFF != ''"
+
+  liveness-test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+      - uses: technote-space/get-diff-action@v1
+        id: git_diff
+        with:
+          SUFFIX_FILTER: |
+            .go
+            .mod
+            .sum
+      - name: build image
+        run: |
+          make build-docker-local-simapp      
```

Comment:
Maybe this is reason enough to push an image @marbar3778. Not blocking I think though.

Reply:
possibly, when I get around to doing updates I can test how long this takes compared to just building each time. If we want multiple of these at varying lengths then it makes sense. 

I think one of the highest priority for ci is reducing the overall time it takes. I havent had time to look into this but will soon

Reply:
Sounds good. Note, this only takes a few min (<5).